decidim-core/app/cells/decidim/amendable/amendments_list/show.erb

```diff
@@ -0,0 +1,10 @@
+<div class="section">
+  <h2 class="section-heading">
+    <%= t("section_heading", scope: "decidim.amendments.amendable", count: model.count) %>
+  </h2>
+  <div class="row small-up-1 medium-up-2 card-grid">
+    <% model.each do |emmendation| %>
+      <%= card_for emmendation, context: { has_footer: false }, has_footer: false , show_footer_actions: false %>
```

Comment:
Do we need to pass is as context & non-context?

Reply:
I'm not sure if this file is being used. Maybe theres been a mistake with all the rebases/merges going around. I'll get back to you on monday.

Reply:
This file isnt used anymore. I've made a mistake while merging/rebasing.